tidb-cloud/connect-to-tidb-cluster.md

```diff
@@ -117,7 +117,7 @@ To connect to your TiDB cluster via VPC peering, perform the following steps:
 
 To connect to your TiDB cluster using SQL shell, perform the following steps:
 
-1. Navigate to the **Clusters** page.
+1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page and choose your target project on the top of the left navigation bar.
 
 2. Locate your cluster, click **Connect** in the upper-right corner of the cluster area, and select the **Web SQL Shell** tab in the connection dialog.
 
```

Comment:
L126 The UI is "Enter password:"
<img width="625" alt="image" src="https://user-images.githubusercontent.com/60599231/207010826-562308f2-ea4a-440c-ac4e-daddb05d9094.png">